keyboardvisibilityevent/src/main/java/net/yslibrary/android/keyboardvisibilityevent/KeyboardVisibilityEvent.java

```diff
@@ -79,7 +75,14 @@ public void onGlobalLayout() {
                         activityRoot.getWindowVisibleDisplayFrame(r);
 
                         int screenHeight = activityRoot.getRootView().getHeight();
-                        int heightDiff = screenHeight - r.height();
+
+                        int statusBarHeight  = 0;
+                        int resourceId = activityRoot.getResources().getIdentifier("status_bar_height", "dimen", "android");
```

Comment:
I'm wondering if this works as expected in the split window environment.
Would be great if this can be done with windowInsets, but I don't think we can get it without `View.onApplyWindowInsets`.

Reply:
FYI `.getIdentifier("status_bar_height"` is a fragile implementation, chris banes recommends against using it https://youtu.be/_mGDMVRO3iE?t=1171